mobify/platform-scaffold
# AMP-53: share styles

Setup sharing styles to work with container styles. 

 **JIRA**: https://mobify.atlassian.net/browse/AMP-43
 **Linked PRs**: n/a

## Changes
- Add sharing styles in global styles
- Set up styles in container

## How to test-drive this PR
- `npm install`
- Run `npm run dev`
- Preview: http://localhost:3000/
- Make sure that styles are in `<style amp-custom>`
